src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -0,0 +1,7 @@
+databaseChangeLog:
+  - include:
+      file: news/000001_create_table.sql
```

Comment:
I like that you used liquidate wrapper, but do we really need sql based scripts that may possibly rely on some concrete RDMS dialect?